java-connectors/kafka-connect-azure-eventhubs/src/main/java/io/lenses/java/streamreactor/connect/azure/eventhubs/source/AzureEventHubsSourceTask.java

```diff
@@ -87,4 +94,21 @@ public void stop() {
     ofNullable(eventHubsKafkaConsumerController)
         .ifPresent(consumerController -> consumerController.close(closeTimeout));
   }
+
+  /**
+   * Returns output topics (if specified in config) or throws {@link ConfigException}.
+   *
+   * @param azureEventHubsSourceConfig task configuration
+   * @return output topics list
+   */
+  private List<String> getOutputTopicsFromConfig(
+      AzureEventHubsSourceConfig azureEventHubsSourceConfig) {
+    Kcql kcql = KcqlConfigPort.parseMultipleKcqlStatementsPickingOnlyFirst(
+        azureEventHubsSourceConfig.getString(AzureEventHubsConfigConstants.KCQL_CONFIG));
+    List<String> outputTopics = Arrays.stream(kcql.getTarget().split(",")).collect(Collectors.toList());
```

Comment:
This is confusing:

Consider these kcql statements
```
INSERT into a,b,c SELECT * FROM X;
```
This is not valid SQL. Sure KCQL is not SQL92 even, but why would you build such a functionality?

Reply:
I removed this code, thanks!